core/src/com/po/fuck/model/Core.java

```diff
@@ -16,6 +16,7 @@ public class Core implements Updatable {
     static { // TODO: remove
         forceInit(Destructable.class);
         forceInit(All.class);
+        // Constants.saveToProperties();
```

Comment:
What is this ?

Reply:
Saving constants to files. As it was used to create .properties , left it for possible changes (like splitting)

Reply:
I mean commented code